app/src/main/java/me/hufman/androidautoidrive/carapp/music/views/BrowseView.kt

```diff
@@ -87,16 +92,24 @@ class BrowseView(val states: List<RHMIState>, val musicController: MusicControll
 			pushBrowsePage(null, stateId)
 		}
 		// show the content for the page that we are showing
-		val currentPage = pageStack.last()
```

Comment:
Perhaps keep this `val currentPage` in place for the nonnull guarantee, and then add a `this.currentPage = currentPage` after.

Reply:
Yeah that would be cleaner, reverted these changes and added that member var assignment to the currentPage